fluidos_model_orchestrator/model/candidate_generation/__init__.py

```diff
@@ -4,7 +4,7 @@
 import torch.nn.functional as F
 import json
 import random  # TODO remove
-import importlib_resources
+import pkg_resources  # noqa
```

Comment:
I'm assuming you're removing importlib due to the package conflict right? The whole PR looks good to me in any case :-)

Reply:
Yes, unfortunately it seems importlib_resources is somehow not loaded properly. I opened an issue to remind the technical debt of using pkg_resources instead.